lib/autosave.coffee

```diff
@@ -9,6 +10,8 @@ module.exports =
 
   subscriptions: null
 
+  provideControls: -> dontSaveIf
```

Comment:
What do you think about providing an object with a method so we can add other methods without breaking major version compatibility with the existing service?